### DISCUSSION TITLE
MULTIMEDIA_LINK question

### DISCUSSION THREAD

**Discussion by devouges** [FamilySearch/GEDCOM#255] [Q&A] [answered]
A multimedia_link will point to a multimedia_record.

The multimedia_record can have a list of FILE entries.

The CROP entry in MULTIMEDIA_LINK will be the same for all the FILE entries ?

This does not seem right to me.

Opinions anyone ?

**Comment by fisharebest** [comment, marked as answer]
I raised this issue earlier.  Perhaps on the mailing list that existed before this repository?

The result was this addition to the specification:

> If the superstructure links to a MULTIMEDIA_RECORD that includes multiple FILE sub‐
structures, the CROP applies to the first FILE to which it can apply, namely the first ex‐
ternal file with a defined pixel unit.
It is recommended that CROP be used only with a single-FILE MULTIMEDIA_RECORD

**Reply by devouges** [reply]
Wow !   Not very....  I don't know..... Just seems to me like there should be a CROP for each file.

My logical thinking I guess......   Thanks for the answer..... At least, it does make some kind of sense.
